src/components/post/CommonPostCard.tsx

```diff
@@ -141,11 +187,11 @@ export default function PostCard(props: props) {
         <CardActions style={{ padding: "4px 8px" }}>
           <Box sx={{ flexGrow: 1 }}></Box>
 
-          {/* <Grow in={!hiddenPostDetail} style={{ transformOrigin: "0 0 0" }}>
-            <Button onClick={() => setOpenSnackBar(true)} variant="contained">
-              Join
+          <Grow in={!hiddenPostDetail} style={{ transformOrigin: "0 0 0" }}>
+            <Button onClick={() => joinPost()} variant="contained">
+              {!isUserJoin ? "Join" : "Cancel"}
             </Button>
```

Comment:
Button still be the old design.
<img width="176" alt="image" src="https://user-images.githubusercontent.com/62459518/226545413-a895e380-310f-4245-b673-1732a44f6ad3.png">

Reply:
Actually, there is no redesign on the index page in figma; however, ive just solved it.

Reply:
I think I leave a comment that design is the same except the box shadow and button however I think I might forgot to put the text on figma. My bad.